docker-compose.yml

```diff
@@ -15,18 +15,18 @@ services:
       - "7777:7777"
     environment:
       - RUN_DEV_SERVER=True
-      - PORT=7777
-      - HOST=0.0.0.0
       - DEBUG=True
+      - PROTOCOL=https
+      - HOST=0.0.0.0
+      - PORT=7777
       - VERSION_PATH=/version.json
       - SECRET_KEY=secret_key_change_me
-      - SESSION_COOKIE_NAME=lando-ui.dev
-      - SESSION_COOKIE_DOMAIN=
-      - SESSION_COOKIE_SECURE=False
+      - SESSION_COOKIE_NAME=lando.mozilla.org
```

Comment:
I've been looking at it like the settings in this file are the defaults for the development environment that way someone can get up and running super easy.

I think it would be helpful also if you added the keys for the OIDC variables that are needed that way it isn't a mystery what they need to add in their overrides file, like so:

```
     - OIDC_DOMAIN=dev_account.auth0.com
     - OIDC_CLIENT_ID=change_me_in_docker_compose_overrides
     - OIDC_CLIENT_SECRET=change_me_in_docker_compose_overrides
```